core/src/main/java/org/geysermc/geyser/network/UpstreamPacketHandler.java

```diff
@@ -308,10 +326,10 @@ public PacketSignal handle(MovePlayerPacket packet) {
 
     @Override
     public PacketSignal handle(ResourcePackChunkRequestPacket packet) {
+        session.setRequestedPacks(true);
```

Comment:
This would probably falsely (not) trigger if a pack is already loaded on a client - since it wouldnt request resource packs from the server then